File: android/app/src/main/java/com/sdust/iotroniks/data/DataManagerHelper.java
package com.sdust.iotroniks.data;

import android.support.annotation.NonNull;

import java.util.List;

public interface DataManagerHelper {

  interface LoadIncidentsCallback {
    void onIncidentsLoaded(List<Incident> incidents);
  }

  void getIncidents(@NonNull LoadIncidentsCallback loadIncidentsCallback);

}

File: android/app/src/main/java/com/sdust/iotroniks/data/DataManagerHelper.kt
package com.sdust.iotroniks.data

interface DataManagerHelper {

  interface LoadIncidentsCallback {
    fun onIncidentsLoaded(incidents: List<Incident>)
  }

  interface LoadOccurrencesCallback {
    fun onOccurrencesLoaded(incidents: List<Incident>)
  }

  fun getIncidents(loadIncidentsCallback: LoadIncidentsCallback)


}
